### PR TITLE
Implants! Now extra-capitalist

### PR DESCRIPTION
**Pull request by Superhats**
## About The Pull Request
Toolset implants may now be purchased from the DeForest import store for around 500 credits each.

## Why It's Good For The Game
It's already possible to order a variety of MOD modules and engineering tools. This adds a relatively expensive alternative to acquire toolset implants if science is understaffed or materials are otherwise lacking. 

## Proof Of Testing
if it runs it runs

## Changelog
:cl:VrazzleDazzle
DeForest has expanded their import catalog to offer a selection of cybernetic implants.
/:cl:

### CODE REVIEW

**Comment by YakumoChen** [comment]
Hi I added implants to DeForest

The implants I chose for DeForest were primarily made to be best suited to doctors. Toolset implants might be better added to a different vendor.

**Comment by projectkepler-ru** [comment]
> Hi I added implants to DeForest
> 
> The implants I chose for DeForest were primarily made to be best suited to doctors. Toolset implants might be better added to a different vendor.
> 
> EDIT: Oh, Skyrat removed the implants I had added before. I might just add them again.

yeah it was a paxilmaniac PR, feel free

File: modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm [hunk 1]
```diff
@@ -180,6 +180,27 @@
...
 	cost = PAYCHECK_LOWER * 2
 
+// Cybernetics and other things that go inside you
+
+/datum/armament_entry/company_import/deforest/implant
+	subcategory = "Cybernetics"
+
+/datum/armament_entry/company_import/deforest/implant/surgery
+	item_type = /obj/item/organ/internal/cyberimp/arm/surgery
+	cost = PAYCHECK_COMMAND * 10
```
Comment (projectkepler-ru) [inline modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm:190]: Honestly, they don't need to cost a 1000, there's a lot worse you can do eitherway

**Comment by Superhats** [comment]
I could halve the price. My reasoning is that they're still a full set of tools which are much faster than advanced surgery tools, so the price should be hefty to encourage going through R&D instead of ordering. 

As for the vendor, I'm not sure which would be more fitting, since none of the others seem like the type to sell cybernetics. Kahraman, maybe?

**Review by projectkepler-ru — APPROVED** [review]

**Comment by ReturnToZender** [comment]
> Either way the toolsets are now half-price, and still under DeForest since it seems weird for something like the mining/frontier vendors to be selling implants.

Please be sure to note this in the body of your PR as well

**Review by Majkl-J — APPROVED** [review]
Looks good, thank you for the work! Now let's hope I dont forget to click merge

**Pull request merged**